lib/generators/shopify_app/add_app_uninstalled_job/templates/app_uninstalled_job.rb.tt

```diff
@@ -0,0 +1,21 @@
+class AppUninstalledJob < ActiveJob::Base
+  extend ShopifyAPI::Webhooks::Handler
+
+  class << self
+    def handle(topic:, shop:, body:)
+      perform_later(topic: topic, shop_domain: shop, webhook: body)
+    end
+  end
+
+  def perform(topic:, shop_domain:, webhook:)
+    shop = Shop.find_by(shopify_domain: shop_domain)
+
+    if shop.nil?
+      logger.error("#{self.class} failed: cannot find shop with domain '#{shop_domain}'")
```

Comment:
I realize you are probably boosting these from templates, but this is a good opportunity for improvement 😄 😬

Reply:
Ok I just added a raise `StandardError` I thought I remember you saying to raise `NotFound` error? I couldn't find it anywhere maybe you said a different error, my bad for forgetting.

Reply:
Alternatively, you could use [find_by!](https://api.rubyonrails.org/v7.0.4/classes/ActiveRecord/FinderMethods.html#method-i-find_by-21) instead of (find_by) and raise an error automagically by the power of rails. We'd lose the logging with that approach though